src/tex/optimize.cpp

```diff
@@ -74,6 +74,11 @@ auto transparent_alpha_required(const Texture &file, const Settings &sets) -> bo
     return is_cubemap && uncompressed && bad_alpha;
 }
 
+[[nodiscard]] auto has_opaque_alpha(const TexMetadata &info) noexcept -> bool
+{
+    return info.GetAlphaMode() == DirectX::TEX_ALPHA_MODE::TEX_ALPHA_MODE_OPAQUE;
+};
+
```

Comment:
DirectXTex also provides [`IsAlphaAllOpaque`](https://github.com/microsoft/DirectXTex/blob/e7ab102d5e8d584115ea8a04ec218817f548c624/DirectXTex/DirectXTexImage.cpp#L800)

We should use it too

Reply:
Done.